utils/build/docker/php/common/tag_value.php

```diff
@@ -11,6 +11,10 @@ function error(){
 	error();
 }
 
+
+$rootSpan = \DDTrace\root_span();
+$rootSpan->meta["http.route"] = '/tag_value/{tag_value}/{status_code}';
```

Comment:
For my knowledge, why this change fixes the issue ?

Reply:
Sort of. There are integrations for major frameworks that add this automatically.

But yes, if http.route isn't set, API security doesn't work. I've been told by Anil that the backend ignores any schemas sent if http.route isn't set anyway. So this is very much a conscious change.